README.md

```diff
@@ -462,6 +463,28 @@ to live.
 
 Note: `chronos-sync.rb` does not delete jobs by default. You can pass the `--delete-missing` flag to `chronos-sync.rb` to remove jobs. Alternatively, you can manually remove it using the API or web UI.
 
+## Mesos Framework Authentication
+
+To enable the credentials in Chronos:
+
+* Run mesos-master with the credentials flag and the path to file with authorized users + their secrets.
+`--credentials=/path/to/credential/file`
+
+> Each line in the file should be a principal space separated with their password.
+```bash
+$ cat /path/to/credential/file
+principal secret
+principal2 secret2
+```
+
+* Run chronos with the `--mesos_authentication_principal` flag and optionally with the
```

Comment:
"Run Chronos with `--mesos_authentication_principal` set to a Mesos-authorized principal. For Mesos' built-in CRAM-MD5 authentication, you must also provide `--mesos_authentication_secret_file` pointing to a file containing your authentication secret.